.changeset/five-kangaroos-march.md

```diff
@@ -0,0 +1,5 @@
+---
+'@astrojs/compiler': minor
```

Comment:
Also pinging @natemoo-re

Reply:
I think it's a `minor` because we expose **a new API** to the browser version of the compiler. This means it's a new feature - already tested/working - exposed to an existing medium.

Reply:
Makes sense, thanks for your review!